kore/src/Log.hs

```diff
@@ -104,7 +105,8 @@ data LogMessage = LogMessage
 
 instance Entry LogMessage where
     entrySeverity LogMessage{severity} = severity
-    oneLineDoc (LogMessage{severity}) = prettySeverity severity
+    oneLineDoc LogMessage{message} = Pretty.pretty message
+    oneLineJson LogMessage{message} = JSON.toJSON message
```

Comment:
Do we also get the severity as a context in the json log?
I thought this would require 
```
oneLineContextJson LogMessage{severity} = JSON.object [ "context" .= JSON.toJSON [show severity] ]` or something similar...